### PR TITLE
Vagrant support

### PR DESCRIPTION
**Pull request by filipgolonka**
@Toxantron 

Added Vagrant support with documentation - easier to contribute with delivered environment, more or less ready to work :)

### CODE REVIEW

**Comment by Toxantron** [comment]
A virtual environment is a good idea. I personally prefer _docker_ for speed and boot time, so I'll build one for docker as well. I don't mind having vagrant as well unless you are fine using docker instead.

File: doc/Vagrant.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+```bash
+./vendor/bin/doctrine orm:schema-tool:create
+./vendor/bin/doctrine orm:generate-proxies
+```
+
+Copy prepared configuration:
+
+```bash
+
```
Comment (Toxantron) [inline doc/Vagrant.md]: Is this intended to be blank? I think you missed the copy command here from the docs.

**Comment by Toxantron** [comment]
@filipgolonka could you check the empty line in the docs? If this is intended I will merge it. ;-)

**Comment by filipgolonka** [comment]
@Toxantron done, I forget about filling in command for copying configuration, now it's ready.

**Comment by Toxantron** [comment]
I expected something like this, just didn't want to mess with your code.
What do you think if we change _sample-config.php_ to match the _vagrant-config.php_? I don't want to maintain two configs in the future and changing the sample config is fine with me.

**Comment by filipgolonka** [comment]
agree, I want just to confirm - do you want to have stuff from `vagrant-config.php` merged into `sample-config.php`? So then only `sample-config.php` will stay in the repository?

**Comment by Toxantron** [comment]
Yes. As far as I can see the only differ in DB credentials and your vagrant values are intuitive enough to serve as a sample as well.

**Comment by Toxantron** [comment]
I will then use the same values for a docker image.

**Comment by filipgolonka** [comment]
done

**Comment by Toxantron** [comment]
Perfect. Thank you very much. This is a great addition and hopefully makes it easier to contribute.

**Comment by Toxantron** [comment]
As promised I added docker support.

**Pull request merged**